docs/software-architecture-and-design.md

```diff
@@ -533,9 +531,7 @@ Although we want to display information about the [AI](#ai) in the [Visualizer](
 A [DrawFunction](#draw_functions) is essentially a function that tells the [Visualizer](#visualizer) _how_ to draw something. When created, [DrawFunctions](#draw_functions) use [lazy-evaluation](https://www.tutorialspoint.com/functional_programming/functional_programming_lazy_evaluation.htm) to embed the data needed for drawing into the function itself. What is ultimately produced is a function that the [Visualizer](#visualizer) can call, with the data to draw (and the details of how to draw it) already included. This function can then be sent over the Observer system to the [Visualizer](#visualizer). The [Visualizer](#visualizer) can then run this function to perform the actual draw operation.
 
 # Estop
-`Estop` allows us to quickly and manually command robots to stop what they are doing, typically through a stateful switch connected to an Arduino. The switch state is communicated to the [Backend](#backend). When `Estop` is in a `STOP` state, the backend overrides the [Primitives](#primitives) sent to the robot to an `Estop primitive`. when it is in a `PLAY` state, primitives are communicated normally.
-
-Since in many cases we run our system without physical robots, the `Estop` system is not enabled until the [Backend](#backend) receives a [Robot Status](#robot-status) message specifically asking for `Estop` to be enabled. However, once it is enabled, `Estop` cannot be disabled. 
+`Estop` allows us to quickly and manually command physical robots to stop what they are doing, typically through a stateful switch connected to an Arduino. The switch state is communicated to the [Backend](#backend). When `Estop` is in a `STOP` state, the backend overrides the [Primitives](#primitives) sent to the robot to an `Estop primitive`. when it is in a `PLAY` state, primitives are communicated normally.
```

Comment:
we should say something about this applying to wIfibackend and radiobackend only

Reply:
changed